action.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Install PocketIC server'
+description: 'Install the PocketIC server at a particular version'
+inputs:
+  pocket-ic-server-version:
+    description: >
+      The PocketIC server version to install.
+      If "latest" is provided, the latest version will be installed.
+    default: 'latest'
+
+runs:
+  using: composite
+  steps:
+    - name: Install PocketIC server
+      shell: sh
+      run: |
+        if [ "${{ inputs.pocket-ic-server-version }}" = "latest" ]; then
+          export POCKET_IC_VERSION="6.0.0"
```

Comment:
Why not just set default to `6.0.0` and you can get rid all this if else lines?

Reply:
I'm not sure what exactly you have in mind since we're mapping "latest" to "6.0.0" and everything else stays as is.

Reply:
Why can't you have this?
```
inputs:
  pocket-ic-server-version:
    description: >
      The PocketIC server version to install.
      If "latest" is provided, the latest version will be installed.
    default: '6.0.0'
    ```

Reply:
Makes sense! Thanks for the detailed suggestion.